app/models/setting.js

```diff
@@ -9,7 +9,7 @@ export default Model.extend(ValidationEngine, {
     title: attr('string'),
     description: attr('string'),
     logo: attr('string'),
-    cover: attr('string'),
+    cover_image: attr('string'),
```

Comment:
Good question. There is no reason, it's just inconsistent. Server uses camel case for our model schema and settings is a model. We should actually make this consistent before releasing the beta.